ceph.spec.in

```diff
@@ -839,7 +839,7 @@ rm -rf %{buildroot}
 %{_sbindir}/ceph-create-keys
 %{_sbindir}/rcceph
 %dir %{_libexecdir}/ceph
-%{_libexecdir}/ceph/ceph_common.sh
+%{_libdir}/ceph/ceph_common.sh
```

Comment:
maybe we should generate `ceph-osd.conf` and `ceph-osd@.service` at the build/configure time from .in files. see `configure_file()` calls `src/CMakeLists.txt` for examples. so these scripts can work with different `-DCMAKE_INSTALL_LIBEXECDIR=/usr/lib` settings. or we need to fix it in `ceph.spec.in` at install time, so these init scripts on fedora/rhel distros are able to find these helper scripts.